packages/@react-native-windows/cli/src/runWindows/runWindows.ts

```diff
@@ -143,12 +145,38 @@ async function runWindows(
         totalMem: totalmem(),
         diskFree: getDiskFreeSpace(__dirname),
         cpus: cpus().length,
+        project: await getAnonymizedProjectName(config.root),
       },
     });
     telemetryClient.flush();
   }
 }
 
+export async function getAnonymizedProjectName(
```

Comment:
why is this in runwindows? This would make more sense as part of creating a new project

Reply:
Mainly just because run-windows is the only thing using this. A project for something this small that's not shared seems like more overhead than its worth, but I can move this to a different file.

Reply:
no I meant why is it being collected as part of run windows instead of as part of creating a new project?

Reply:
Building is more useful than init to measure active usage, which is the particular bit that I'm curious about. I think this could be useful in both places though.

Reply:
I'm not convinced, let's discuss offline. Thanks!

Reply:
Sure, we can discuss offline, but init is kind of already per-project and doesn't really allow collecting some of the data that I'm interested in. I.e. it allows measuring creation of new projects, but not the population of active projects.